.editorconfig

```diff
@@ -1,7 +1,13 @@
 root = true
 
 [*]
+charset = utf-8
 indent_style = tab
 indent_size = 2
 trim_trailing_whitespace = true
-insert_final_newline = true
\ No newline at end of file
+insert_final_newline = true
+
+# json uses 2 space indents
+[*.{json,jsonc,json5,yml,yaml}]
+indent_style = space
+indent_size = 2
```

Comment:
Hm no, `pnpm format` will scream at you ^^

We use tabs for everything, only the `pnpm-lock.yml` files are ignored

Reply:
Oh i thought this meant json should use 2 spaces instead of tabs https://github.com/huggingface/huggingface.js/pull/55#issuecomment-1459093271

make sense now. Though I didn't get any complains from pnpm format. Thanks for looking out